src/pages/account/settings/index.tsx

```diff
@@ -0,0 +1,172 @@
+        />
+      ),
+      binding: (
+        <FormattedMessage
+          id="accountandsettings.menuMap.binding"
+          defaultMessage="Account Binding"
+        />
+      ),
+      notification: (
+        <FormattedMessage
+          id="accountandsettings.menuMap.notification"
+          defaultMessage="New Message Notification"
+        />
+      ),
+    };
+    this.state = {
+      mode: 'inline',
+      menuMap,
+      selectKey: 'base',
+    };
+  }
+
+  componentDidMount() {
+    const { dispatch } = this.props;
+    dispatch({
+      type: 'accountAndsettings/fetchCurrent',
+    });
+    window.addEventListener('resize', this.resize);
+    this.resize();
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize', this.resize);
+  }
+
+  getMenu = () => {
+    const { menuMap } = this.state;
+    return Object.keys(menuMap).map(item => <Item key={item}>{menuMap[item]}</Item>);
+  };
+
+  getRightTitle = () => {
+    const { selectKey, menuMap } = this.state;
+    return menuMap[selectKey];
+  };
+
+  selectKey = (key: SettingsStateKeys) => {
+    this.setState({
+      selectKey: key,
+    });
+  };
+
+  resize = () => {
+    if (!this.main) {
+      return;
+    }
+    requestAnimationFrame(() => {
+      if (!this.main) {
+        return;
+      }
+      let mode: 'inline' | 'horizontal' = 'inline';
+      const { offsetWidth } = this.main;
+      if (this.main.offsetWidth < 641 && offsetWidth > 400) {
+        mode = 'horizontal';
+      }
+      if (window.innerWidth < 768 && offsetWidth > 400) {
+        mode = 'horizontal';
+      }
+      this.setState({
+        mode,
+      });
+    });
+  };
+
+  renderChildren = () => {
+    const { selectKey } = this.state;
+    switch (selectKey) {
+      case 'base':
+        return <BaseView />;
+      case 'security':
+        return <SecurityView />;
+      case 'binding':
+        return <BindingView />;
+      case 'notification':
+        return <NotificationView />;
+      default:
+        break;
+    }
+
+    return null;
+  };
+
+  render() {
+    const { currentUser } = this.props;
+    if (!currentUser.userid) {
+      return '';
+    }
+    const { mode, selectKey } = this.state;
+    return (
+      <GridContent>
+        <div
+          className={styles.main}
+          ref={ref => {
+            if (ref) {
+              this.main = ref;
+            }
+          }}
+        >
+          <div className={styles.leftMenu}>
+            <Menu
+              mode={mode}
+              selectedKeys={[selectKey]}
+              onClick={({ key }) => this.selectKey(key as SettingsStateKeys)}
+            >
+              {this.getMenu()}
+            </Menu>
+          </div>
+          <div className={styles.right}>
+            <div className={styles.title}>{this.getRightTitle()}</div>
+            {this.renderChildren()}
+          </div>
+        </div>
+      </GridContent>
+    );
+  }
+}
+
+export default Settings;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of default exports is forbidden](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)